test/fixture/OUTPUT.md

```diff
@@ -1,5 +1,47 @@
 # Automd built-in generator fixtures
 
+## Table of Contents
+
+<!-- automd:toc -->
+
+- [Table of Contents](#table-of-contents)
+- [Heading test](#heading-test)
+  - [sub heading 1](#sub-heading-1)
+  - [sub heading 2](#sub-heading-2)
+- [heading with link](#heading-with-link)
+- [heading with footnote](#heading-with-footnote)
```

Comment:
This link is broken. The ast for this node is

```json
[
  {
    "type": "h2",
    "children": [
      "heading with footnote",
      {
        "type": "footnoteReference"
      }
    ]
  },
  {
    "type": "footnoteDefinition",
    "children": [
      {
        "type": "p",
        "children": ["this is a footnote"]
      }
    ]
  }
]
```

The slug generated by this plugin is `#heading-with-footnote` while github generates `#heading-with-footnote1`